src/components/NcAppNavigationItem/NcAppNavigationItem.vue

```diff
@@ -438,7 +456,7 @@ export default {
 	emits: [
 		'update:menuOpen',
 		'update:open',
-		'update:title',
```

Comment:
This should stay. We still emit title.

Reply:
Well, this is for documentation only.
Doesn't have any effect for vue2 and should most likely not be documented as it's deprecated now

Reply:
Hm, fine with me. Vue3 will complain, but since it'll be a major anyway, it's probably ok.

Reply:
Perfect, one reason to make sure this is removed in the next major! :stuck_out_tongue_closed_eyes: